src/main/java/me/totalfreedom/totalfreedommod/ChatManager.java

```diff
@@ -51,10 +51,10 @@ private void handleChatEvent(AsyncPlayerChatEvent event)
         // Strip color from messages
         message = ChatColor.stripColor(message);
 
-        // Truncate messages that are too long - 100 characters is vanilla client max
-        if (message.length() > 100)
+        // Truncate messages that are too long - 250 characters is vanilla client max
+        if (message.length() > 250)
```

Comment:
As per [the Minecraft wiki page](http://minecraft.gamepedia.com/1.11), the maximum length should be 256.